msix/appxmanifest.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:uap5="http://schemas.microsoft.com/appx/manifest/uap/windows10/5"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
+  xmlns:desktop4="http://schemas.microsoft.com/appx/manifest/desktop/windows10/4"
+  xmlns:desktop6="http://schemas.microsoft.com/appx/manifest/desktop/windows10/6"
+  xmlns:iot2="http://schemas.microsoft.com/appx/manifest/iot/windows10/2" IgnorableNamespaces="mp uap uap3 uap5 rescap desktop desktop4 desktop6 iot2">
+
+  <Identity Name="JohannesSchindelin.Git" Publisher="CN=Johannes Schindelin, O=Johannes Schindelin, L=Koeln, S=North Rhine Westphalia, C=DE" Version="1.0.0.0" ProcessorArchitecture="neutral"/>
+
+  <Properties>
+    <DisplayName>Git</DisplayName>
+    <PublisherDisplayName>Johannes Schindelin</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
+
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17134.0" MaxVersionTested="10.0.17134.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.18362.0" />
+    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.29231.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
+  </Dependencies>
+
+  <Resources>
+    <Resource Language="en-us"/>
+  </Resources>
+
+  <Applications>
+    <Application Id="GitBash" Executable="Git\git-bash.exe" EntryPoint="Windows.FullTrustApplication">
+      <uap:VisualElements
+        DisplayName="Git Bash"
+        Description="Git Bash"
+        BackgroundColor="transparent"
+        Square150x150Logo="Images\Square150x150Logo.png"
```

Comment:
Obviously we will want to change this image to something that looks like Git for Windows' logo ;-)

Also, is it possible to use an `.svg` instead? We have that: https://github.com/git-for-windows/build-extra/blob/HEAD/git-for-windows.svg (it's not only the logo but also the word "git", but that's easily adjusted).

Reply:
We can't use SVGs, they don't allow that. Visual Studio has a feature where you load a PDF and then it auto-scales it and creates all the different resolutions that are needed from that. So that is what I did now, starting from the SVG you linked to.